_maps/map_files/Heliostation/Heliostation.dmm

```diff
@@ -70944,6 +70933,28 @@
 	},
 /turf/open/floor/plasteel/white,
 /area/security/prison)
+"kpw" = (
+/obj/docking_port/stationary{
+	dir = 4;
+	dwidth = 12;
+	height = 18;
+	id = "emergency_home";
+	name = "BoxStation emergency evac bay";
+	width = 32
+	},
+/obj{
+	name = "---Merge conflict marker---"
```

Comment:
what does this mean

Reply:
I honestly dont know. But i did have a merge conflict here that i had to fix. So that might have something to do with it. Ill look at it when i get home.